src/swarm/neo/util/Formatter.d

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+
+    Wrapper around Ocean's ocean.text.convert.Formatter.sformat used to
+    avoid usage of the deprecated overload that accepts a sink returning
+    size_t in case overload accepting sink returning void is present (since
+    ocean v2.8.
+
```

Comment:
Maybe a note that this module could be removed and revert all code to simply using ocean's `Formatter`, in the next major?